### PR TITLE
refs #1648 fixed %try-module docs and improved error handling, other …

### PR DESCRIPTION
**Pull request by davidnich**
…minor doc reference fixes

### CODE REVIEW

**Comment by davidnich** [comment]
@omusil24 please propagate to develop after merging, thx

**Review by omusil24 — APPROVED** [review]

**Comment by omusil24** [comment]
Waiting for @pavelkveton review.

**Review by pavelkveton — APPROVED** [review]

**Pull request merged**
